content/docs/concepts/traffic-management/index.md

```diff
@@ -417,8 +377,8 @@ spec:
             subset: v1
 {{< /text >}}
 
-In the example, note that under `spec`, which lists the specifications of the
-network resource, `hosts` lists the virtual service's hosts. In this case, the
+In the example under `spec`, the resource specification,
```

Comment:
```suggestion
In the example, under `spec`, the `hosts` resource specification
```

Reply:
`spec` is the "resource specification", `hosts` is just a part of it.

Reply:
```suggestion
In the example, under `spec`,
```